app/models/user.rb

```diff
@@ -20,6 +20,10 @@ def scoped_user_groups
     end
   end
 
+  def user_group?
```

Comment:
to reveal the intent of this method, I might call it something like `can_select_user_group?` and put it somewhere outside of the user model. Like maybe in a helper? Usually I see view-specific methods in helper modules rather than in ActiveRecord classes.